aten/src/ATen/native/native_functions.yaml

```diff
@@ -6371,6 +6371,7 @@
 - func: zeros(SymInt[] size, *, ScalarType? dtype=None, Layout? layout=None, Device? device=None, bool? pin_memory=None) -> Tensor
   dispatch:
     CompositeExplicitAutograd: zeros_symint
+    NestedTensorCPU, NestedTensorCUDA: zeros_nested_symint
```

Comment:
This is kind of smelly, idk

Reply:
Oh hmm is the idea that this is bad because its not immediately obvious that only strided NestedTensors go through this path?

Reply:
We're only going to add as many of these as it takes to get rid of strided NestedTensors specific backward formulas though.